File: mobile-app/app/CinemaApplication/app/src/main/java/com/example/luulac/cinemaapplication/data/models/showtimes/FilmScheduleModel.java
package com.example.luulac.cinemaapplication.data.models.showtimes;

import java.util.List;

public class FilmScheduleModel {
    public String date;
    public String day;
    public String dateOfWeek;
    public List<DateScheduleModel> dateScheduleModels;

    public FilmScheduleModel() {
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getDay() {
        return day;
    }

    public void setDay(String day) {
        this.day = day;
    }

    public String getDateOfWeek() {
        return dateOfWeek;
    }

    public void setDateOfWeek(String dateOfWeek) {
        this.dateOfWeek = dateOfWeek;
    }

    public List<DateScheduleModel> getDateScheduleModels() {
        return dateScheduleModels;
    }

    public void setDateScheduleModels(List<DateScheduleModel> dateScheduleModels) {
        this.dateScheduleModels = dateScheduleModels;
    }
}

File: mobile-app/api/CinemaBookingSolution/CinemaBookingCore/Data/Models/FilmScheduleModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using CinemaBookingCore.Data.Entities;

namespace CinemaBookingCore.Data.Models
{
    public class FilmScheduleModel
    {
        public String Date { get; set; }
        public String Day { get; set; }
        public String DateOfWeek { get; set; }
        public List<DateScheduleModel> DateScheduleModels { get; set; }
    }
}
